app/src/main/java/com/aaronjwood/portauthority/network/Wireless.java

```diff
@@ -186,6 +186,38 @@ public int getInternalWifiSubnet() {
         return netmask;
     }
 
+
+    public int getInternalWifiIpAddressAsInt() {
```

Comment:
RFC 1918 excludes those ranges, so shouldn't be a problem, at least for LAN scanning.